repos/fdbt-site/src/pages/api/createCaps.ts

```diff
@@ -11,105 +12,150 @@ import {
     removeExcessWhiteSpace,
 } from '../../utils/apiUtils/validator';
 import { updateSessionAttribute } from '../../utils/sessions';
+import { isADayOrLonger } from '../createCaps';
+import { daysOfWeek } from '../../../src/constants';
 
 export interface InputtedCap {
     name: string | undefined;
     price: string | undefined;
     durationAmount: string | undefined;
     durationUnits: string | undefined;
+    type: string | undefined;
+    startDay: string | undefined;
 }
 
-export const validateAndFormatCapInputs = (
-    inputtedCaps: InputtedCap[],
-    cappedProductName: string | undefined,
-): { errors: ErrorInfo[]; caps: Cap[] } => {
+export const isADayOfTheWeek = (input: string | undefined): boolean => {
```

Comment:
If this function is being used outside of this file (which is it), it needs to be moved into a different file e.g. apiUtils. Also, it needs 2 unit tests writing.